cypress/e2e/gui.cy.js

```diff
@@ -0,0 +1,145 @@
+      cy.get('button[class^="Messenger_openCloseButton"')
+       .click()
+      cy.get('div[class^="Messenger_form"')
+       .should('be.visible')
+      cy.get('button[class^="Messenger_openCloseButton"')
+       .click()
+      cy.get('div[class^="Messenger_form"')
+       .should('not.exist')
+    })
+    it('It makes sure all messenger`s fields are mandatory and the first one is focused', () => {
+      cy.get('button[class^="Messenger_openCloseButton"')
+       .click()
+      cy.get('input[id="messenger-name"]')
+       .should('have.focus')
+       .and('have.prop', 'required', true)
+      cy.get('input[id="email"]')
+       .should('be.visible')
+       .and('have.prop', 'required', true)
+      cy.get('textarea[id="message"]')
+       .should('be.visible')
+       .and('have.prop', 'required', true)
+    })
+    it('It shows and hides a success message when successfully submitting the messenger form', () => {
+      cy.get('button[class^="Messenger_openCloseButton"')
+       .click()
+      cy.get('input[id="messenger-name"]')
+       .type('Joe')
+      cy.get('input[id="email"]')
+       .type('email@test.com')
+      cy.get('textarea[id="message"]')
+       .type('Hello there!')
+      cy.get('button[type="submit"]').click()
+      cy.get('div[class*="Messenger_success"]')
+       .should('exist')
+       .and('have.text', 'Your message has been sent.')
+      cy.get('div[class*="Messenger_success"]')
+       .should('not.exist')
+    })
+
+    it.only('It clears all the messenger`s form fields when filling them, closing the messenger, and opening it again', () => {
+      cy.get('button[class*="Messenger_openCloseButton"')
+       .click()
+      cy.get('div[class*="Messenger_form"')
+       .should('be.visible')
+      cy.get('input[id="messenger-name"]')
+       .type('Joe')
+      cy.get('input[id="email"]')
+       .type('email@test.com') 
+      cy.get('textarea[id="message"]')
+       .type('Hello there!')
+      cy.get('button[class*="Messenger_openCloseButton"')
+       .click()
+      cy.get('button[class*="Messenger_openCloseButton"')
+       .click()
+       cy.get('input[id="messenger-name"]')
+       .should('have.value', '')
+      cy.get('input[id="email"]')
+       .should('have.value', '')
+      cy.get('textarea[id="message"]')
+       .should('have.value', '')
+    })
+
+    it('It shows the Company name and Action columns and hides the ID, Industry, Number of Employees, and Size columns in a mobile viewport', () => {
+      cy.viewport('iphone-6')
+      cy.get('th').
+      contains('Company name').should('be.visible')
+      cy.get('th')
+      .contains('Action').should('be.visible')
+      cy.get('th')
+      .contains('ID').should('not.be.visible')
+      cy.get('th')
+      .contains('Industry').should('not.be.visible')
+      cy.get('th')
+      .contains('Number of employees').should('not.be.visible')
+      cy.get('th')
+      .contains('Size').should('not.be.visible')
+    });    
+  });
```

Comment:
```suggestion
  });

```

Recomenda-se deixar uma linha em branco ao final de todos os arquivos.

Isso faz com que ferramentas como o GitHub não exibam o ícone de `No newline at end of file` durante o
processo de _code review_.

Além disso, isso é uma convensão e boa prática a ser seguida.

> O mesmo vale para qualquer outro arquivo.